ansible-ipi-install/roles/installer/templates/etc-chrony.conf.j2

```diff
@@ -11,7 +11,12 @@ spec:
       security:
         tls: {}
       timeouts: {}
+{% if ((release_version[0]|int == 4) and (release_version[2]|int > 5)) %}
+      version: 3.1.0
+{% endif %}
+{% if ((release_version[0]|int == 4) and (release_version[2]|int < 6)) %}
       version: 2.2.0
+{% endif %}
```

Comment:
The `filesystem: root` below only works with ignition 2.x - it won't work with 3.x as that option has been removed

Reply:
Thanks, addressed via https://github.com/openshift-kni/baremetal-deploy/pull/530